src/terminal/node/terminal-backend-module.ts

```diff
@@ -22,13 +22,20 @@ export default new ContainerModule(bind => {
 @injectable()
 class TerminalExpressContribution implements BackendApplicationContribution {
     private terminals: Map<number, any> = new Map()
-    private logs: string[] = []
+
+    getShellExecutablePath() {
```

Comment:
Does this method need to be visible from outside of the class? Probably no, since the class is not even exported. Please use the `private` modifier.